tests/unit/index.test.js

```diff
@@ -1,5 +1,49 @@
-import { describe, expect, it } from "vitest";
+import { describe, expect, it, vi } from "vitest";
+import { shortAddr } from '/src/shared/lib/utils';
```

Comment:
Using absolute paths like `/src/shared/lib/utils` in import statements is not standard and might not work as expected without additional configuration, since Node.js does not natively resolve such paths for modules.
It’s better to use relative paths to make sure your imports work as expected everywhere.